docs.local/planning/golems-v2-branding-plan.md

```
@@ -0,0 +1,989 @@
+but EmailGolem isn't installed. Want me to set it up?
+Run: golems addon install email-golem"
+```
+
+---
+
+## Part 3: Notification Channel Migration
+
+> **STATUS: DROPPED** - Root cause identified as iOS 26 beta APNs bug (affects ALL apps, not just Telegram).
+> Keep Telegram as-is. Revisit when iOS 26 stable fixes push notifications.
+
+### Problem (RESOLVED)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)